### DISCUSSION TITLE
Collapse and command-function

### DISCUSSION THREAD

**Discussion by kotarokun** [home-assistant/frontend#9582] [Automations / scripts / scenes]
Hello,

I have two things on my wish list that would make programming clearer. 
First, it would be great if you could collapse the choose points to improve the overview of automations with multiple choose elements. 

Secondly, I would like to know if it is possible to add comment functions next to or in the automation overview? When you look at the automations later, you sometimes don't know exactly what is being done at one point or another.

I hope these wishes meet with understanding.

Maze

Translated with www.DeepL.com/Translator (free version)

**Comment by karwosts** [comment]
Relevant PR: #17239

**Reply by kotarokun** [reply]
Thanks so much for this feature
